index.js

```diff
@@ -56,8 +56,26 @@ function fromString (str) {
 }
 
 function floatize (lonlat) {
-  const lon = parseFloat(lonlat.lon || lonlat.lng || lonlat.longitude)
-  return {lon: lon, lat: parseFloat(lonlat.lat || lonlat.latitude)}
+  const lon = parseFloatWithAlternates([lonlat.lon, lonlat.lng, lonlat.longitude])
```

Comment:
@evansiroky since we're not transpiling this library we need to still use `var`.